README.md

```diff
@@ -1,33 +1,13 @@
-_Fork this project and send us a pull request_
+1. for install requirements run command:
+    sudo pip install -r requirements.txt
 
-Write a simple python webservice that returns the objects & manipuates the data found here [http://www.unisport.dk/api/sample/](http://www.unisport.dk/api/sample/).
+2. for make test run command:
+    python manage.py test
 
+3. for create DATABASE run command:
+    python manager.py syncdb
```

Comment:
Syncdb is deprecated since 1.7
https://docs.djangoproject.com/en/1.8/ref/django-admin/#syncdb